### PR TITLE
Django version

### PR DESCRIPTION
**Pull request by zarino**
Heavily rebased version of @struan’s https://github.com/mysociety/neighbourhood-warmth/pull/10, with extra stuff added on top (like Bootstrap Sass pipeline and some basic views).

### CODE REVIEW

File: neighbourhood/mixins.py [hunk 1]
```diff
@@ -2,10 +2,13 @@ class TitleMixin:
...
-            return f"{self.page_title} | {self.site_title}"
-        else:
-            return f"{self.site_title}"
+        try:
+            if self.page_title:
+                return f"{self.page_title} | {self.site_title}"
+        except AttributeError:
+            pass
+
+        return f"{self.site_title}"
```
Comment (struan) [inline neighbourhood/mixins.py:27]: a slightly better approach might be:

```python
if hasattr(self, "page_title"):
    return f"{self.page_title} | {self.site_title}"
else:
    return f"{self.site_title}"
```

File: neighbourhood_warmth/urls.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from neighbourhood import views
+
+urlpatterns = [
+    path("", views.HomePageView.as_view(), name="home"),
+    path("search/", views.DisambiguationView.as_view(), name="search_results"),
+    path("street/<slug:street>/", views.StreetView.as_view(), name="street"),
+    path("street/<slug:street>/join/", views.StreetJoinView.as_view(), name="street_join"),
+    path("street/<slug:street>/actions/", views.StreetActionsView.as_view(), name="street_actions"),
+    path("street/<slug:street>/update/", views.StreetUpdateView.as_view(), name="street_update"),
+    path("area/<slug:gss>/", views.AreaView.as_view(), name="area"),
```
Comment (struan) [inline neighbourhood_warmth/urls.py:27]: nothing seems to link to this.

**Review by struan — APPROVED** [review]
This all looks fine, modulo a few small comments

File: neighbourhood_warmth/urls.py (hunk 2 above)
Comment (zarino) [reply neighbourhood_warmth/urls.py:27]: Yeah, not yet!

**Pull request merged**